src/Scrollable/index.jsx

```diff
@@ -19,7 +19,8 @@ const Scrollable = ({ children, ...props }) => {
     },
   );
   return (
-    <div {...props} className={className}>
+    // eslint-disable-next-line jsx-a11y/no-noninteractive-tabindex
+    <div {...props} className={className} tabIndex="0">
```

Comment:
[curious] Can you walk me through the thinking on making `Scrollable` focusable? I'm not sure whether it makes sense for it to be focusable, but curious to hear the rationale.

Reply:
@PKulkoRaccoonGang After chatting with Jeff Witt, this `tabIndex` behavior is actually recommended. However, we should avoid the eslint error here by adding an appropriate `role` attribute to the `div`. I think `role="group" would probably make sense here and should resolve the error?